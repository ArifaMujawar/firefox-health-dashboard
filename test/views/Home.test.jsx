import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../../src/Home';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
