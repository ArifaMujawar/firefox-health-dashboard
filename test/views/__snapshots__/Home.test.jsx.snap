// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="DashboardPage-root-4 Home-root-1"
>
  <div
    className="DashboardPage-title-5"
  >
    <h1
      style={
        Object {
          "display": "inline",
        }
      }
    >
      Firefox health
      <small
        className="DashboardPage-subtitle-6"
      >
        Tracking metrics for Firefox products
      </small>
    </h1>
    <div
      style={
        Object {
          "float": "right",
          "padding": "0 1rem 0 0",
        }
      }
    >
      <a
        href="/"
        title="Home"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
      </a>
      <a
        href="/android"
        title="Android Performance"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M16 1H8C6.34 1 5 2.34 5 4v16c0 1.66 1.34 3 3 3h8c1.66 0 3-1.34 3-3V4c0-1.66-1.34-3-3-3zm-2 20h-4v-1h4v1zm3.25-3H6.75V4h10.5v14z"
          />
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
        </svg>
      </a>
      <a
        href="/quantum/64"
        title="Quantum 64 Performance"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M21 2H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h7v2H8v2h8v-2h-2v-2h7c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H3V4h18v12z"
          />
        </svg>
      </a>
      <a
        href="/power"
        title="Power Usage"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M15.67 4H14V2h-4v2H8.33C7.6 4 7 4.6 7 5.33v15.33C7 21.4 7.6 22 8.33 22h7.33c.74 0 1.34-.6 1.34-1.33V5.33C17 4.6 16.4 4 15.67 4zM11 20v-5.5H9L13 7v5.5h2L11 20z"
          />
        </svg>
      </a>
      <a
        href="/playback"
        title="Playback"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M17 10.5V7c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1v10c0 .55.45 1 1 1h12c.55 0 1-.45 1-1v-3.5l4 4v-11l-4 4z"
          />
        </svg>
      </a>
      <a
        href="https://github.com/mozilla-frontend-infra/firefox-health-dashboard/"
        title="Source Code"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root-7"
          focusable="false"
          role="presentation"
          style={
            Object {
              "margin": "0 .2rem 0 .2rem",
              "verticalAlign": "baseline",
            }
          }
          viewBox="0 0 24 24"
        >
          <path
            d="M0 0h24v24H0V0z"
            fill="none"
          />
          <path
            d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0l4.6-4.6-4.6-4.6L16 6l6 6-6 6-1.4-1.4z"
          />
        </svg>
      </a>
    </div>
  </div>
  <div
    className="Home-links-2"
  >
    <a
      href="/android"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-7"
        focusable="false"
        role="presentation"
        style={
          Object {
            "margin": "0 .2rem 0 .2rem",
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M16 1H8C6.34 1 5 2.34 5 4v16c0 1.66 1.34 3 3 3h8c1.66 0 3-1.34 3-3V4c0-1.66-1.34-3-3-3zm-2 20h-4v-1h4v1zm3.25-3H6.75V4h10.5v14z"
        />
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
      </svg>
       Android
    </a>
    <a
      href="/quantum/32"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-7"
        focusable="false"
        role="presentation"
        style={
          Object {
            "margin": "0 .2rem 0 .2rem",
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M21 2H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h7v2H8v2h8v-2h-2v-2h7c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H3V4h18v12z"
        />
      </svg>
       Quantum 32bit
    </a>
    <a
      href="/quantum/64"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-7"
        focusable="false"
        role="presentation"
        style={
          Object {
            "margin": "0 .2rem 0 .2rem",
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M21 2H3c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h7v2H8v2h8v-2h-2v-2h7c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm0 14H3V4h18v12z"
        />
      </svg>
       Quantum 64bit
    </a>
    <a
      href="/battery"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-7"
        focusable="false"
        role="presentation"
        style={
          Object {
            "margin": "0 .2rem 0 .2rem",
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M15.67 4H14V2h-4v2H8.33C7.6 4 7 4.6 7 5.33v15.33C7 21.4 7.6 22 8.33 22h7.33c.74 0 1.34-.6 1.34-1.33V5.33C17 4.6 16.4 4 15.67 4zM11 20v-5.5H9L13 7v5.5h2L11 20z"
        />
      </svg>
       Power Usage
    </a>
    <a
      href="/playback"
    >
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root-7"
        focusable="false"
        role="presentation"
        style={
          Object {
            "margin": "0 .2rem 0 .2rem",
            "verticalAlign": "baseline",
          }
        }
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M17 10.5V7c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1v10c0 .55.45 1 1 1h12c.55 0 1-.45 1-1v-3.5l4 4v-11l-4 4z"
        />
      </svg>
       Playback
    </a>
  </div>
  <div
    className="Home-netlify-3"
  >
    <a
      href="https://www.netlify.com"
    >
      <img
        alt="Netlify"
        src="https://www.netlify.com/img/global/badges/netlify-dark.svg"
      />
    </a>
  </div>
</div>
`;
